misc/python/materialize/cloudtest/k8s/environmentd.py

```diff
@@ -148,6 +148,8 @@ def generate_stateful_set(self) -> V1StatefulSet:
             "--adapter-stash-url=postgres://root@cockroach.default:26257?options=--search_path=adapter",
             "--storage-stash-url=postgres://root@cockroach.default:26257?options=--search_path=storage",
             "--internal-sql-listen-addr=0.0.0.0:6877",
+            "--internal-persist-pubsub-listen-addr=0.0.0.0:6879",
+            "--persist-pubsub-service-url=http://persist-pubsub",
```

Comment:
Seems to come from here.
```suggestion
            "--persist-pubsub-url=http://persist-pubsub",
```

Reply:
whoops! missed that when I did the last renaming, thanks for the pointer